tests/v1/distributed/test_external_lb_dp.py

```
@@ -131,6 +144,9 @@ def default_server_args():
         "128",
         "--enforce-eager",
     ]
+    if GPU_MEMORY_UTILIZATION is not None:
+        server_args.extend(["--gpu-memory-utilization", str(GPU_MEMORY_UTILIZATION)])
```

Ruff (0.16.3)

[error] Undefined name `server_args`

(F821)